securetar/__init__.py

```diff
@@ -318,6 +320,11 @@ def _add_stream(
         tar.offset += size_of_inner_tar + padding_size
 
         tar_info.size = size_of_inner_tar
+        if padding:
+            tar_info.pax_headers = {
+                **tar_info.pax_headers,
+                "_securetar.plaintext_size": str(size_of_inner_tar - len(padding) - 16),
```

Comment:
Can we make `_securetar.plaintext_size` a constant somewhere?

Reply:
Btw, what does 16 relate to here? :thinking:

Reply:
Oops, 16 is the cipher block size, we should use the constant instead of a magic number 👍